src/seed.ts

```diff
@@ -56,7 +56,16 @@ createConnection({
     user.ownedNotes = [note];
     await connection.manager.save([user, note, revision]);
     const foundUser = await connection.manager.findOne(User);
+    if (!foundUser) {
+      throw new Error('Could not find freshly seeded user. Aborting.');
+    }
     const foundNote = await connection.manager.findOne(Note);
+    if (!foundNote) {
+      throw new Error('Could not find freshly seeded note. Aborting.');
+    }
+    if (!foundNote.alias) {
+      throw new Error('Could not find alias of freshly seeded note. Aborting.');
```

Comment:
These should be combined

Reply:
They intentionally are not combined, as they are different errors.
I could merge all seed-errors into one, but then the error message will be less helpful...